site/content/en/docs/Examples/_index.md

```diff
@@ -18,11 +18,11 @@ Access the Kubernetes Dashboard running within the minikube cluster:
 
 Once started, you can interact with your cluster using `kubectl`, just like any other Kubernetes cluster. For instance, starting a server:
 
-`kubectl run hello-minikube --image=k8s.gcr.io/echoserver:1.4 --port=8080`
+`kubectl create deployment hello-minikube --image=k8s.gcr.io/echoserver:1.4`
 
-Exposing a service as a NodePort
+Exposing a service
 
-`kubectl expose deployment hello-minikube --type=NodePort`
+`kubectl expose deployment hello-minikube --port=8080`
```

Comment:
please keep the NodePort, because the it is needed for minikube to create a url. please change it to
``` kubectl expose deployment hello-minikube --type=NodePort --port=8080```

Reply:
done